sci-electronics/librepcb/librepcb-1.1.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2009-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="https://librepcb.org/"
+SRC_URI="https://download.librepcb.org/releases/${PV}/${P}-source.zip"
+
+S="${WORKDIR}/${PN}-${PV/_/-}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="opencascade qt6"
+
+BDEPEND="
+	app-arch/unzip
+	!qt6? ( dev-qt/linguist-tools:5 )
+	qt6? ( dev-qt/qttools:6[linguist] )"
+
+RDEPEND="
+	dev-cpp/muParser:=
+	!qt6? (  dev-libs/quazip:=[qt5(+)]
+			dev-qt/qtconcurrent:5
+			dev-qt/qtcore:5
+			dev-qt/qtgui:5
+			dev-qt/qtnetwork:5[ssl]
+			dev-qt/qtopengl:5
+			dev-qt/qtprintsupport:5
+			dev-qt/qtquickcontrols2:5
+			dev-qt/qtsql:5
+			dev-qt/qtsvg:5
+			dev-qt/qtwidgets:5
+			dev-qt/qtxml:5
+	)
+	qt6? (  dev-libs/quazip:=[qt6(+)]
+			dev-qt/qtbase[concurrent,dbus,gui,network,opengl,ssl,sql,sqlite,test,widgets,xml] )
+	opencascade? ( sci-libs/opencascade:= )
+	sys-libs/zlib
+	virtual/opengl"
+
+DEPEND="${RDEPEND}
+	!qt6? ( dev-qt/qttest:5 )
+	qt6? ( dev-qt/qtbase:6[test] )"
```

Comment:
```suggestion
```

qtbase:6 includes the modules needed for testing without the test use flag.

Reply:
thank you, fixed